03_launch_mgmt_cluster.sh

```diff
@@ -14,6 +14,18 @@ export IRONIC_HOST="${CLUSTER_URL_HOST}"
 export IRONIC_HOST_IP="${CLUSTER_PROVISIONING_IP}"
 export REPO_IMAGE_PREFIX="quay.io"
 
+# -k is for keepalived
+BMO_IRONIC_ARGS=(-k)
```

Comment:
It would be nice if you'd explicitly declare this as an array (declare -a) before use

Reply:
Thanks, fixed